packages/engine/Specs/Scene/PostProcessStageCollectionSpec.js

```diff
@@ -381,110 +381,152 @@ describe(
-        expect(rgba[2]).toEqual(0);
-        expect(rgba[3]).toEqual(255);
+      it("uses ACES tonemapping", function () {
+        if (!scene.highDynamicRangeSupported) {
+          return;
+        }
+
+        const fs =
+          "void main() { \n" +
+          "    out_FragColor = vec4(4.0, 0.0, 0.0, 1.0); \n" +
+          "} \n";
+        scene.primitives.add(new ViewportPrimitive(fs));
+
+        scene.postProcessStages.tonemapper = Tonemapper.ACES;
+
+        const colorWithoutHdr = [255, 0, 0, 255];
+
+        expect(scene).toRender(colorWithoutHdr);
+        scene.highDynamicRange = true;
+        expect(scene).toRenderAndCall(function (rgba) {
+          expect(rgba).not.toEqual(black);
+          expect(rgba).not.toEqual(colorWithoutHdr);
+          expect(rgba[0]).withContext("r").toBeGreaterThan(0);
+          expect(rgba[1]).withContext("g").toEqual(0);
+          expect(rgba[2]).withContext("b").toEqual(0);
+          expect(rgba[3]).withContext("a").toEqual(255);
+        });
+        scene.highDynamicRange = false;
       });
-      scene.highDynamicRange = false;
-    });
 
-    it("uses ACES tonemapping", function () {
-      if (!scene.highDynamicRangeSupported) {
-        return;
-      }
-
-      const fs =
-        "void main() { \n" +
-        "    out_FragColor = vec4(4.0, 0.0, 0.0, 1.0); \n" +
-        "} \n";
-      scene.primitives.add(new ViewportPrimitive(fs));
-
-      scene.postProcessStages.tonemapper = Tonemapper.ACES;
-
-      expect(scene).toRender([255, 0, 0, 255]);
-      scene.highDynamicRange = true;
-      expect(scene).toRenderAndCall(function (rgba) {
-        expect(rgba).not.toEqual([0, 0, 0, 255]);
-        expect(rgba).not.toEqual([255, 0, 0, 255]);
-        expect(rgba[0]).toBeGreaterThan(0);
-        expect(rgba[1]).toEqual(0);
-        expect(rgba[2]).toEqual(0);
-        expect(rgba[3]).toEqual(255);
+      it("uses PBR Neutral tonemapping", function () {
+        if (!scene.highDynamicRangeSupported) {
+          return;
+        }
+
+        const fs =
+          "void main() { \n" +
+          "    out_FragColor = vec4(4.0, 0.0, 0.0, 1.0); \n" +
+          "} \n";
+        scene.primitives.add(new ViewportPrimitive(fs));
+
+        scene.postProcessStages.tonemapper = Tonemapper.PBR_NEUTRAL;
+
+        const colorWithoutHdr = [255, 0, 0, 255];
+        expect(scene).toRender(colorWithoutHdr);
+        scene.highDynamicRange = true;
+        expect(scene).toRenderAndCall(function (rgba) {
+          expect(rgba).not.toEqual(black);
+          expect(rgba).not.toEqual(colorWithoutHdr);
+          expect(rgba[0]).withContext("r").toBeGreaterThan(0);
+          // TODO: I'm not 100% sure how to verify this. The other tests only modify the red channel...
+          expect(rgba[0]).withContext("r").toEqual(253);
+          expect(rgba[1]).withContext("g").toEqual(149);
+          expect(rgba[2]).withContext("b").toEqual(149);
+          expect(rgba[3]).withContext("a").toEqual(255);
```

Comment:
I need a little help here. I'm not sure what exactly _should_ be the expected result here. The only other tonemap spec that's different is the modified Reinhard one and that changes the starting `fragColor` instead.